app/models/label/LabelValidationTable.scala

```diff
@@ -48,10 +48,10 @@ class LabelValidationTable (tag: slick.lifted.Tag) extends Table[LabelValidation
   def canvasWidth = column[Int]("canvas_width", O.NotNull)
   def startTimestamp = column[java.sql.Timestamp]("start_timestamp", O.NotNull)
   def endTimestamp = column[java.sql.Timestamp]("end_timestamp", O.NotNull)
-  def isMobile = column[Boolean]("is_mobile", O.NotNull)
+  def source = column[String]("source", O.NotNull)
```

Comment:
As of right now, the `source` column is actually nullable! I think there are two options here:
1. Make the column nullable.
1. Keep the column as non-null, and use a "sentinel value" for the old rows where we don't know the source for sure. It could be something as simple as "N/A" or "Old" or "Old - Unsure", lmk if you think of something better.

I think that a sentinel value is my preferred method, because we know that it won't be null ever in the future. It keeps all future code simple, and only minorly complicates the evolutions script.

You would basically create the column as nullable, update the column for all existing entries such that they are no longer null, then add the `NOT NULL` constraint. Note that I would not use a default value here.